tests/network/localnet/liblocalnet.py

```
@@ -77,6 +78,8 @@ def localnet_vm(
     networks: list[Network],
     interfaces: list[Interface],
     network_data: cloudinit.NetworkData | None = None,
+    pod_anti_affinity: bool = True,
```

Ruff (0.15.10)

[warning] Boolean-typed positional argument in function definition

(FBT001)